README.md

```diff
@@ -24,7 +24,12 @@ to include a lot of other stuff e.g. the orchard library.
 
 1. Update `depend/zcash` with the latest tagged version of `zcashd`
 2. Update `Cargo.toml` versions to match the versions used by the latest tagged version of `zcashd`, and its dependencies
-3. Publish a new release
+3. For dependencies that are shared with Zebra (but not `zcashd`), match the latest version in Zebra's [Cargo.lock](https://github.com/ZcashFoundation/zebra/blob/main/Cargo.lock):
+    - use `cargo tree --invert <crate>` to see if the crate is from `zcash_script` or another dependency
+    - see the list in [Cargo.toml](https://github.com/ZcashFoundation/zcash_script/blob/master/Cargo.toml#L69)
+4. For new dependencies with a leading zero in their version (`0.x.y`), use a `>=` dependency [to make them automatically upgrade to match Zebra's dependencies](https://doc.rust-lang.org/cargo/reference/resolver.html#semver-compatibility)
+5. Check all open PRs to see if they can be merged before the release
+6. Publish a new release
```

Comment:
Do we want to say run cargo release with no-publish first to commit any automatic changes?
 
```suggestion
5. Run `cargo-release` with `--no-publish` to commit any automatic changes
6. Check all open PRs to see if they can be merged before the release
7. Publish a new release
```

Reply:
I'd like to do it in the opposite order, in case the open PRs change anything about the release or `Cargo.toml`?